ts/store/reducers/authentication.ts

```diff
@@ -176,7 +176,7 @@ function matchWithIdp<O>(
   whenWithoutIdp: O,
   whenWithIdp: (state: AuthenticationStateWithIdp) => O
 ): O {
-  if (state.kind === "LoggedOutWithoutIdp") {
+  if (state.kind === "LoggedOutWithoutIdp" || state.kind === undefined) {
```

Comment:
Unfortunately this part is very weak. But your comment gives me an idea to improve:
The authentication store section is wiped when the user does logout
https://github.com/pagopa/io-app/blob/796dd83ec5b352edc6c2cf7b3c3abf2479ea56f8/ts/store/reducers/index.ts#L166
So that section could be undefined

I improved it by setting the initial state, so `state.kind` won't be `undefined`
here the commit 885b936768c6a969a4e0411cab670bee686de153